admin/models/pupil-status-code.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose')
+mongoose.Promise = global.Promise
+const Schema = mongoose.Schema
+
+const PupilStatusCode = new Schema({
+  status: {
```

Comment:
This is actually a description.  Do you think the name reflects this?   When we move to SQL I'm more used to fields like this being called `statusDesc`...